src/rpc/blockchain.cpp

```diff
@@ -2269,6 +2273,7 @@ static RPCHelpMan dumptxoutset()
         },
         RPCExamples{
             HelpExampleCli("dumptxoutset", "utxo.dat")
+    + HelpExampleCli("dumptxoutset", "\"utxo.db\" \"sqlite\"")
```

Comment:
Formatting nit-pick:
```suggestion
            HelpExampleCli("dumptxoutset", "utxo.dat") +
            HelpExampleCli("dumptxoutset", "\"utxo.db\" \"sqlite\"")
```